{"uuid": "8679cb49-8162-4069-b825-c70e966b8be8", "children": ["c4800fab-a532-4d25-af80-1285d13a314f"], "befores": [{"name": "login", "status": "passed", "start": 1594170929230, "stop": 1594170942278}], "afters": [{"name": "login::0", "status": "broken", "statusDetails": {"message": "ConnectionResetError: [WinError 10054] 远程主机强迫关闭了一个现有的连接。\n", "trace": "  File \"d:\\program files\\python37\\lib\\site-packages\\allure_commons\\_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"d:\\program files\\python37\\lib\\site-packages\\_pytest\\fixtures.py\", line 796, in _teardown_yield_fixture\n    next(it)\n  File \"E:\\CDATA\\自动化测试\\Cdata_part11\\tests\\GponSfu\\conftest.py\", line 19, in login\n    ont_del(tn, PonID, OnuID, SN)\n  File \"E:\\CDATA\\自动化测试\\Cdata_part11\\src\\Gpon\\ont_auth.py\", line 1067, in ont_del\n    tn.write(b\"interface gpon 0/0\" + b\"\\n\")\n  File \"d:\\program files\\python37\\lib\\telnetlib.py\", line 290, in write\n    self.sock.sendall(buffer)\n"}, "start": 1594170978426, "stop": 1594170978428}], "start": 1594170929230, "stop": 1594170978428}